### PR TITLE
feat: add logger for print

### PR DESCRIPTION
**Pull request by Yongle-Fu**

### CODE REVIEW

**Comment by Sunbreak** [comment]
Thanks for your contribution. I'll try to keep up with when available

**Comment by Sunbreak** [comment]
`quick_blue_linux` is added. Could you rebase onto the master?

**Comment by Yongle-Fu** [comment]
done

File: quick_blue_platform_interface/lib/method_channel_quick_blue.dart [hunk 1]
```diff
@@ -13,6 +15,16 @@ class MethodChannelQuickBlue extends QuickBluePlatform {
     _message_connector.setMessageHandler(_handleConnectorMessage);
   }
 
+  QuickLogger? _logger;
+  @override
```
Comment (Sunbreak) [inline quick_blue_platform_interface/lib/method_channel_quick_blue.dart:19]: New line

File: quick_blue/lib/quick_blue.dart [hunk 2]
```diff
@@ -34,7 +36,11 @@ QuickBluePlatform get _platform {
 }
 
 class QuickBlue {
-  static Future<bool> isBluetoothAvailable() => _platform.isBluetoothAvailable();
+  static void setLogger(QuickLogger logger) =>
+      QuickBluePlatform.instance.setLogger(logger);
```
Comment (Sunbreak) [inline quick_blue/lib/quick_blue.dart:40]: _platform.setLogger

**Review by Sunbreak — APPROVED** [review]

File: quick_blue_linux/lib/quick_blue_linux.dart [hunk 3]
```diff
@@ -24,6 +24,9 @@ class QuickBlueLinux extends QuickBluePlatform {
     }
   }
 
+  @override
+  void setLogger(QuickLogger logger) {}
```
Comment (Sunbreak) [inline quick_blue_linux/lib/quick_blue_linux.dart:28]: Impl in quick_blue_linux

**Pull request merged**